mtgrasp.smk

```diff
@@ -202,14 +202,18 @@ rule extract_seq:
          assemblies=rules.select_length.output
      output:
          query_out=current_dir + "{library}/mito_filtering_output/k{k}_kc{kc}-scaffolds.blast-mt_db.fa",
-         ref_out=current_dir + "{library}/mito_filtering_output/k{k}_kc{kc}-ref.fa"
+         ref_out=current_dir + "{library}/mito_filtering_output/k{k}_kc{kc}-ref.fa",
      benchmark:
          current_dir + "{library}/benchmark/k{k}_kc{kc}.extract_seq.benchmark.txt"
      params:
-         ref_fasta=config["ref_path"]
+         ref_fasta=config["ref_path"],
+         ref_outdir=current_dir + "{library}/blast/mito_refs",
+         ref_config = current_dir + "{library}/blast/k{k}_kc{kc}-ntjoin_ref_config.csv"
      shell:
          "seqtk subseq {input.assemblies} {input.query} > {output.query_out} ; "
-         "seqtk subseq {params.ref_fasta} {input.ref} > {output.ref_out}"
+         "seqtk subseq {params.ref_fasta} {input.ref} > {output.ref_out} &&"
```

Comment:
`&&` shouldn't be needed? Could just use `;`
Ideally, these should be separate rules - this is now doing sort of separate things (extracting sequences and creating the config file for ntJoin). But if it'll take too long to change that, OK to leave as-is